server/mailer/templates/registrationTemplate.ejs

```diff
@@ -0,0 +1,345 @@
+          <path
+            d="M2346 178 c4 -25 3 -39 -2 -35 -6 3 -16 1 -24 -5 -12 -8 -12 -9 3 -4
+  14 4 17 -1 17 -24 0 -28 -3 -30 -35 -30 -19 0 -35 -4 -35 -10 0 -13 7 -13 68
+  5 56 17 77 41 83 95 6 43 -1 50 -41 50 l-40 0 6 -42z m59 -48 c-3 -5 -11 -10
+  -17 -10 -6 0 -17 -8 -24 -17 -12 -16 -13 -15 -14 10 0 24 4 27 31 27 17 0 28
+  -4 24 -10z"
+          />
+          <path
+            d="M2852 165 c-1 -73 4 -91 24 -99 27 -10 54 12 54 44 0 25 -4 30 -25
+  30 -21 0 -25 -5 -26 -32 -1 -21 -5 -11 -10 27 -13 90 -17 96 -17 30z"
+          />
+          <path
+            d="M2880 185 c0 -31 3 -35 25 -35 22 0 25 4 25 35 0 31 -3 35 -25 35
+  -22 0 -25 -4 -25 -35z"
+          />
+          <path
+            d="M1071 199 c-1 -9 13 -36 31 -60 24 -33 33 -40 36 -27 4 19 -19 48
+  -36 48 -6 0 -15 12 -20 28 -6 15 -11 20 -11 11z"
+          />
+          <path
+            d="M310 176 l-35 -21 33 -3 c39 -4 45 -1 40 25 -3 19 -4 19 -38 -1z"
+          />
+          <path
+            d="M2116 164 c-24 -24 -19 -42 17 -64 42 -26 47 -25 47 10 0 28 -3 30
+  -37 31 l-38 2 40 6 c38 6 39 7 18 19 -27 15 -28 15 -47 -4z"
+          />
+          <path
+            d="M1695 150 c3 -5 1 -10 -4 -10 -6 0 -11 -13 -11 -30 0 -25 5 -31 33
+  -38 30 -8 32 -7 35 22 3 22 -3 34 -20 48 -25 19 -43 24 -33 8z"
+          />
+          <path
+            d="M2570 115 c0 -34 4 -45 19 -50 35 -11 51 4 51 50 0 42 0 43 -35 43
+  -35 0 -35 0 -35 -43z"
+          />
+          <path
+            d="M198 142 c-68 -3 -68 -3 -68 -32 l0 -30 63 0 c34 0 67 4 72 8 10 7
+  14 48 5 54 -3 2 -35 2 -72 0z"
+          />
+          <path
+            d="M1168 143 c-24 -6 -23 -50 1 -63 35 -18 51 -12 51 20 0 31 -22 49
+  -52 43z"
+          />
+          <path
+            d="M1852 138 c-13 -13 -17 -68 -4 -68 4 0 18 7 30 16 26 18 29 49 6 58
+  -20 8 -18 8 -32 -6z"
+          />
+          <path
+            d="M52 113 c2 -23 8 -29 35 -31 30 -3 32 -1 31 27 -1 28 -4 31 -35 31
+  -32 0 -34 -2 -31 -27z"
+          />
+          <path
+            d="M600 111 c0 -44 10 -61 35 -61 25 0 35 17 35 61 0 27 -3 29 -35 29
+  -32 0 -35 -2 -35 -29z"
+          />
+          <path
+            d="M1317 133 c-4 -3 -7 -19 -7 -35 0 -20 5 -28 18 -28 26 1 52 25 52 49
+  0 16 -6 21 -28 21 -16 0 -32 -3 -35 -7z"
+          />
+          <path
+            d="M1920 106 c0 -38 20 -52 51 -36 12 7 19 21 19 40 0 28 -3 30 -35 30
+  -33 0 -35 -2 -35 -34z"
+          />
+          <path
+            d="M2190 107 c0 -32 2 -35 35 -39 34 -4 35 -3 35 29 0 20 -5 33 -12 33
+  -7 0 -23 3 -35 6 -21 6 -23 3 -23 -29z"
+          />
+          <path
+            d="M1230 93 c0 -31 2 -33 35 -33 33 0 35 2 35 33 0 30 -2 32 -35 32 -33
+  0 -35 -2 -35 -32z"
+          />
+          <path
+            d="M1760 118 c0 -8 20 -12 48 -11 25 1 5 -22 -21 -25 -39 -5 -33 -24 6
+  -20 28 3 32 7 35 34 3 29 1 31 -32 30 -20 -1 -36 -4 -36 -8z"
+          />
+        </g>
+      </svg>
+      <div>
+        <h1 style="font-weight: 300; text-transform: capitalize">
```

Comment:
do we need all these styles and the svg? IMO a simple email conveying the information the receiver needs is more effective.

Reply:
I didn't have any copy of what the email should say so I took the liberty of adding their logo to make it look and feel a bit more official.

Reply:
let's go with the simplest approach for now. If they want to make it look better we can include Erica/Berfin to design something. This is an internal email, though, so it just needs the information.

Reply:
👍 
[bc9cbcf](https://github.com/AMP-SCZ/dpdash/pull/457/commits/bc9cbcfb5650ebceec9074f06d4e04dcc47fc58f)